receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -339,11 +355,15 @@ func (jr *jReceiver) EmitBatch(batch *jaeger.Batch) error {
 }
 
 func (jr *jReceiver) GetSamplingStrategy(serviceName string) (*sampling.SamplingStrategyResponse, error) {
-	return &sampling.SamplingStrategyResponse{}, nil
+	return jr.agentSamplingManager.GetSamplingStrategy(serviceName)
 }
 
 func (jr *jReceiver) GetBaggageRestrictions(serviceName string) ([]*baggage.BaggageRestriction, error) {
-	return nil, nil
+	br, err := jr.agentSamplingManager.GetBaggageRestrictions(serviceName)
+	if err != nil {
+		return nil, nil
```

Comment:
return err?

Reply:
So baggage restrictions are not actually implemented - refer - https://github.com/jaegertracing/jaeger/blob/c1bc28d829f0b01103aa2957684b3e38a6d6ccbb/cmd/agent/app/configmanager/grpc/manager.go#L50-L53

But we still want to return a valid `200 OK` response. Hence the `return nil, nil`

Reply:
Maybe add a comment to help future devs understand it?